### PR TITLE
Feat:Changes made to fix app-insights issue

### PR DESCRIPTION
**Pull request by shivajihmcts**
JIRA link

https://tools.hmcts.net/jira/browse/RDCC-4614

Change description

RD API app-insight is not generating logs for api.

Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[ x] No

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JsonFeignResponseUtilTest.java [hunk 1]
```diff
@@ -55,12 +52,12 @@ void test_Decode_fails_with_ioException() {
...
-            when(bodyMock.asReader(Charset.defaultCharset())).thenThrow(new IOException());
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
+        //      try {
+        //          when(bodyMock.asInputStream()).thenThrow(new IOException());
+        //          when(bodyMock.asReader(Charset.defaultCharset())).thenThrow(new IOException());
+        //      } catch (IOException e) {
+        //          e.printStackTrace();
+        //      }
```
Comment (AdilOoze) [inline src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JsonFeignResponseUtilTest.java:60]: Why is this being commented out?

**Review by arshinsalim — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JsonFeignResponseUtilTest.java (hunk 1 above)
Comment (shivajihmcts) [reply src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JsonFeignResponseUtilTest.java]: Checked with team, Its unused code so removed

**Review by AdilOoze — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/cwrdapi/util/JavaRecordFeat.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package uk.gov.hmcts.reform.cwrdapi.util;
```
Comment (SujeethKumarAri) [inline src/main/java/uk/gov/hmcts/reform/cwrdapi/util/JavaRecordFeat.java:1]: I have written this class to verify if Java 17 feature works as a POC. This change is not required as part of this ticket

File: src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JavaRecordFeatTest.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.cwrdapi.util;
```
Comment (SujeethKumarAri) [inline src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JavaRecordFeatTest.java:1]: I have written this class to verify if Java 17 feature works as a POC. This change is not required as part of this ticket

**Review by SujeethKumarAri — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JavaRecordFeatTest.java (hunk 3 above)
Comment (shivajihmcts) [reply src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JavaRecordFeatTest.java]: done

**Review by sahitya-desireddy — APPROVED** [review]

**Pull request merged**